www/front_src/src/Resources/Graph/Performance/index.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+
+import {
+  ComposedChart,
+  Area,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Legend,
+  ResponsiveContainer,
+  Tooltip,
+} from 'recharts';
+import filesize from 'filesize';
```

Comment:
we'll probably need to make a choice between filesize and numeral for number conversion (post 20.04)